### DISCUSSION TITLE
Homarr updated to 0.14.1 - lost my previous dashboard

### DISCUSSION THREAD

**Discussion by di5cord20** [tteck/Proxmox#2060] [Help Needed] [answered]
I can see my previous dashboard in /opt/homarr/configs/default.json 
But the vanilla default board loads from a new install.

Any ideas how to restore my previously created dashboard?

**Comment by tteck** [comment]
Have you rebooted the LXC?

**Reply by di5cord20** [reply]
Yes, no change

**Reply by di5cord20** [reply]
Fixed the issue.  Ended up copying the /opt/homarr/configs/default.json to /opt/homarr/data/configs/default.json (thus overwriting the one that existed in that directory) which restored my dashboard.

I raised an issue in the [Homarr Discord](https://discord.com/channels/972958686051962910/1174797640899231764).  Not sure if my issue is a bug on the upgrade or something else.  Will reply here if they share anything that requires a potential update in your script.  As a potential workaround, adding a step in the script to copy all dashboards from /opt/homarr/configs/ to /opt/homarr/data/configs/ might be a potential workaround.

**Reply by tteck** [reply]
sounds like I have the paths messed up

**Reply by tteck** [reply, marked as answer]
fixed
